### ISSUE TITLE
CLI deposit and bond functions can't accept large numbers

### ISSUE THREAD

**Issue by dob** [livepeer/go-livepeer#197]
1 LPT = 10^18 base units of our token. When you go to deposit a reasonable amount such as 10 LPT you receive the following error

```
> 2
Current deposit: 99999999999806
Current token balance: 999999999900000000000000
Enter Deposit Amount - > 100000000000000000000      
ERROR[11-21|13:00:12] Invalid input, expected integer          err="strconv.Atoi: parsing \"100000000000000000000\": value out of range"
> 
```

You receive the same error when attempting to bond().

**Comment by dob** [comment]
The "bonding to yourself" feature within the `-transcoder` CLI also does not parse the big integers correctly and shows this error.

**Comment by yondonfu** [comment]
Should be fixed by #249
